src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -237,20 +240,19 @@ case class JesBackend(actorSystem: ActorSystem)
 
     generateAuthJson(dockerConf, getGcsAuthInformation(workflow)) foreach { content =>
 
-      val path = GcsPath(gcsAuthFilePath(workflow))
-      def upload(prev: Option[Unit]) = connection.storage.uploadJson(path, content)
+      val path = gcsAuthFilePath(workflow).toAbsolutePath(List(connection.gcsFileSystem))
+      def upload(prev: Option[Unit]) = {
+        path.writeAsJson(content)
+        ()
+      }
```

Comment:
Instead of explicitly returning a unit, I think I'd prefer to explicitly set the return type:

```scala
def upload(prev: Option[Unit]): Unit = path.writeAsJson(content)
```